dhis-2/dhis-api/src/main/java/org/hisp/dhis/program/ProgramStage.java

```diff
@@ -345,6 +349,42 @@ public void setDueDateLabel(String dueDateLabel) {
     this.dueDateLabel = dueDateLabel;
   }
 
+  @JsonProperty
+  @JacksonXmlProperty(namespace = DxfNamespaces.DXF_2_0)
+  @PropertyRange(min = 2)
+  public String getProgramStageLabel() {
+    return programStageLabel;
+  }
+
+  @JsonProperty
+  @JacksonXmlProperty(namespace = DxfNamespaces.DXF_2_0)
+  @Translatable(propertyName = "programStageLabel", key = "PROGRAM_STAGE_LABEL")
+  public String getDisplayProgramStageLabel() {
+    return getTranslation("PROGRAM_STAGE_LABEL", getProgramStageLabel());
+  }
+
+  public void setProgramStageLabel(String programStageLabel) {
```

Comment:
could we have this setters, along with the existing ones, be automatically generated?

Reply:
see the reply to Ivo, I don't see Lombok being used for database definition